modules/async-icq/testing/demo-simapp/scripts/protocgen.sh

```diff
@@ -11,7 +11,7 @@ buf generate --template buf.gen.gogo.yaml
 cd ..
 
 # move proto files to the right places
-cp -r github.com/cosmos/ibc-apps/modules/async-icq/v7/interchain-query-demo/x/interquery/types/* x/interquery/types/
+cp -r github.com/cosmos/ibc-apps/modules/async-icq/v8/interchain-query-demo/x/interquery/types/* x/interquery/types/
```

Comment:
QOL for future IBC upgrades
```suggestion
cp -r github.com/cosmos/ibc-apps/modules/async-icq/v*/interchain-query-demo/x/interquery/types/* x/interquery/types/
```